tests/sentry/snuba/metrics/test_snql.py

```diff
@@ -32,17 +32,38 @@
 
 class DerivedMetricSnQLTestCase(TestCase):
     def setUp(self):
+        self.org_id = 666
         self.metric_ids = [0, 1, 2]
+        indexer.bulk_record(
+            {
+                self.org_id: [
+                    "abnormal",
+                    "crashed",
+                    "errored_preaggr",
+                    "errored",
+                    "exited",
+                    "init",
+                    "session.status",
+                    TransactionSatisfactionTagValue.FRUSTRATED.value,
+                    TransactionSatisfactionTagValue.SATISFIED.value,
+                    TransactionSatisfactionTagValue.TOLERATED.value,
+                    TransactionStatusTagValue.CANCELLED.value,
+                    TransactionStatusTagValue.OK.value,
+                    TransactionStatusTagValue.UNKNOWN.value,
+                    TransactionTagsKey.TRANSACTION_SATISFACTION.value,
+                    TransactionTagsKey.TRANSACTION_STATUS.value,
+                ]
+            }
+        )
```

Comment:
We noticed that some tests pass because `resolve_weak` returns `-1` for unknown strings.